common/src/api/external/mod.rs

```diff
@@ -2514,7 +2514,7 @@ pub struct SwitchPortLinkConfig {
     pub mtu: u16,
 
     /// The forward error correction mode of the link.
-    pub fec: LinkFec,
```

Comment:
The internal API got a nice doc update; can we do the same here?

https://github.com/oxidecomputer/dendrite/pull/1038/files#diff-cb166332b35711d4032d1bbea0e1243b00ed0d61e13bd9b9cd63fe39b6bd9df7